packages/pigment-css-react/tsconfig.json

```diff
@@ -9,6 +9,6 @@
     "resolveJsonModule": true,
     "types": ["node", "mocha"]
   },
-  "include": ["src/**/*.tsx", "src/**/*.ts", "src/**/*.jsx", "src/**/*.js", "src/Box/Box.ts"],
+  "include": ["src/**/*.tsx", "src/index.ts", "src/**/*.jsx", "src/**/*.js", "src/Box/Box.ts"],
```

Comment:
When I tried including all .ts files I got this error:
```
      > @pigment-css/react@0.0.6 build /Users/marijanajdova/workspace/pigment-css/packages/pigment-css-react
      > tsup
      
      error TS5055: Cannot write file '/Users/marijanajdova/workspace/pigment-css/packages/pigment-css-react/src/Box.d.ts' because it would overwrite input file.
```
This is why I included only the necessary files. I initially included them because they were not present in the typescript project that we use in the `generateProptypes`. I tried also excluding the .d files, but then the generateProptypes script complained because it couldn't find the Box.d.ts file.

@brijeshb42 what was the original reason for setting up the tsconfig like this?

Reply:
Using this works -

```
"include": ["src/**/*.tsx", "src/**/*.js", "src/**/*.ts"],
"exclude": ["./tsup.config.ts", "src/**/*.d.ts"]
```

Reply:
Updated.